src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.scss

```diff
@@ -0,0 +1,38 @@
+@import 'src/styles/common';
+
+.blastSubmissionResultsContainer {
+  max-width: 1860px;
+  padding: 0 30px $global-padding-bottom 60px;
+}
+
+.listedBlastSubmission:not(:last-child) {
+  margin-bottom: 48px;
+}
+
+.sequenceBoxWrapper {
+  padding: 24px 26px; // TODO: the 26px padding looks like a constant asking to be extracted
+  border: 1px solid $grey;
+}
+
+.sequenceBoxWrapper + .sequenceBoxWrapper {
+  border-top: none;
+}
+
+.resultsSummaryRow {
+  display: grid;
+  grid-template-columns: 5% 62% auto;
```

Comment:
Suspicious numbers. They aren't even enough to contain the phrase `Sequence 1`; imagine what's going to happen to `Sequence 10`.

<img width="1168" alt="image" src="https://user-images.githubusercontent.com/6834224/181047390-40aa48d5-bbde-4a26-a3fc-4eea33e7df59.png">


Plus, did we say we were going to allow users to name their sequences? Do we have any idea how that is going to fit into the boxes?

Reply:
Good point. Something to discuss with Andrea